zerver/lib/import_realm.py

```diff
@@ -94,6 +95,28 @@ def fix_upload_links(data: TableData, message_table: TableName) -> None:
                     if message['rendered_content']:
                         message['rendered_content'] = message['rendered_content'].replace(key, value)
 
+def create_subscription_events(data: TableData, table: TableName) -> None:
+    all_subscription_logs = []
+    for item in data[table]:
+        recipient = Recipient.objects.get(id=item['recipient_id'])
+        if recipient.type == Recipient.STREAM:
+            stream = Stream.objects.get(id=recipient.type_id)
+            user = UserProfile.objects.get(id=item['user_profile_id'])
+
+            # from bulk_add_subscriptions in lib/actions
+            event_last_message_id = Message.objects.aggregate(Max('id'))['id__max']
+            if event_last_message_id is None:
+                event_last_message_id = -1
+            event_time = timezone_now()
```

Comment:
I think we ideally should do this computation of the time and last message ID at the top of the function to avoid repeating it in the loop.